app/config/router-config.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+module.exports = ['$stateProvider', '$urlRouterProvider', routerConfig];
+
+function routerConfig($stateProvider, $urlRouterProvider) {
+  $urlRouterProvider.when('/', '/home');
```

Comment:
$urlRouterProvider should be passed an empty string as the first argument so that way the url automatically defaults to http://localhost:8080/#!/home otherwise I would have to explicitly type /#!/home as the endpoint to change routes.

The beautiful jankiness of angular :D